packages/salesforce/src/Adaptor.js

```diff
@@ -338,9 +338,11 @@ export function create(sObjectName, records) {
 
     return connection
       .create(resolvedSObjectName, resolvedRecords)
-      .then(recordResult => {
-        console.log('Result : ' + JSON.stringify(recordResult));
-        return composeNextState(state, recordResult);
+      .then(result => {
```

Comment:
Shouldn't we use exactly the same structure for a single item?

It still makes perfect sense, and makes documentation and code cleaner (including in job code!)